controller/src/beerocks/master/wfa_ca.h

```diff
@@ -21,6 +21,31 @@ class wfa_ca {
                           std::shared_ptr<beerocks_message::cACTION_HEADER> beerocks_header,
                           ieee1905_1::CmduMessageRx &cmdu_rx, ieee1905_1::CmduMessageTx &cmdu_tx,
                           db &database, task_pool &tasks);
+
+private:
+    enum class eWfaCaStatus : uint8_t { RUNNING, INVALID, ERROR, COMPLETE };
+    static std::string wfa_ca_status_to_string(eWfaCaStatus status);
+
+    enum eWfaCaCommand : uint8_t {
```

Comment:
consider changing to enum class (nitpick)

Reply:
I change it, although it doesn't really matter because the enums definitions are private anyway. and it can't be accessed from outside...

Reply:
Why do you specify the `uint8_t`? This is not a wire format definition, so by specifying `uint8_t` you're just increasing code size, no?